### DISCUSSION TITLE
Sponsored Brand Video Reporting?

### DISCUSSION THREAD

**Discussion by edsel45** [amzn/ads-advanced-tools-docs#309] [General]
How can I discern Sponsored Brand ads to Sponsored Brand Video ads?

**Comment by edsel45** [comment]
I see that Sponsored Brand Videos are available in v2, is there an equivalent version in v3?

`curl --request POST 'https://advertising-api.amazon.com/v2/hsa/campaigns/report' \
--header 'Content-Type: application/json' \
--header 'Authorization: Bearer xxxxxx' \
--header 'Amazon-Advertising-API-ClientId: xxxxxx' \
--header 'Amazon-Advertising-API-Scope: xxxxxx' \
--data-raw '{
"reportDate": "20211102",
"metrics": "viewableImpressions,vtr,vctr,clicks,cost,attributedConversions14d,attributedSales14d",
"creativeType": "video"
}'`

**Reply by phase-sys** [reply]
@edsel45 hi, have you had a solution for discerning sb and sb video in v3?
